src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,289 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.*;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.io.File;
+
+public class Duke {
+    public static final int INDEX_TODO = 5;
+    public static final int INDEX_EVENT = 6;
+    public static final int INDEX_DEADLINE = 9;
+    public static final String SECTION_DIVIDER = "____________________";
+    public static boolean invalidInput = false;
+    private static ArrayList<Task> TaskArray = new ArrayList<>();
```

Comment:
Consider the [variable name convention](https://se-education.org/guides/conventions/java/basic.html#naming) and `TaskArray` is referring to a class or collection.